youtube_dl/extractor/indavideo.py

```diff
@@ -27,42 +35,79 @@ class IndavideoEmbedIE(InfoExtractor):
             'duration': 72,
             'age_limit': 0,
             'tags': ['tánc', 'cica', 'cuki', 'cukiajanlo', 'newsroom'],
-        },
+        }
+    }, {
+        'url': 'http://indavideo.hu/video/Vicces_cica_1',
+        'only_matching': True,
     }, {
         'url': 'http://embed.indavideo.hu/player/video/1bdc3c6d80?autostart=1&hide=1',
         'only_matching': True,
     }, {
         'url': 'http://assets.indavideo.hu/swf/player.swf?v=fe25e500&vID=1bdc3c6d80&autostart=1&hide=1&i=1',
         'only_matching': True,
+    }, {
+        'url': 'http://index.indavideo.hu/video/2015_0728_beregszasz',
+        'only_matching': True,
+    }, {
+        'url': 'http://auto.indavideo.hu/video/Sajat_utanfutoban_a_kis_tacsko',
+        'only_matching': True,
+    }, {
+        'url': 'http://erotika.indavideo.hu/video/Amator_tini_punci',
+        'only_matching': True,
+    }, {
+        'url': 'http://film.indavideo.hu/video/f_hrom_nagymamm_volt',
+        'only_matching': True,
+    }, {
+        'url': 'http://palyazat.indavideo.hu/video/Embertelen_dal_Dodgem_egyuttes',
+        'only_matching': True,
     }]
 
+    @staticmethod
+    def _extract_urls(webpage):
+        return [url for _, url in re.findall(
+            r'<(?:iframe[^>]+src|object[^>]+data)=(["\'])(?P<url>(?:https?:)?//(?:(?:.+?\.)?indavideo\.hu/(?:player/)?video/|assets\.indavideo\.hu/swf/(?:inda)?player\.swf\?.*\b(?:v(?:ID|id))=)[a-zA-Z0-9-_]+)(?:\?|&|\1)',
+            webpage)]
+
     def _real_extract(self, url):
         video_id = self._match_id(url)
 
         video = self._download_json(
             'http://amfphp.indavideo.hu/SYm0json.php/player.playerHandler.getVideoData/%s' % video_id,
             video_id)['data']
 
-        title = video['title']
+        title = video.get('title')
```

Comment:
No, you don't. Title is mandatory meaning you must fail if you can't extract it. Read new extractor tutorial.